src/Reflectivity-VariableBreakpoint/VariableBreakpointTest.class.st

```diff
@@ -328,22 +328,22 @@ VariableBreakpointTest >> testInitialize [
 { #category : #'tests - installation' }
 VariableBreakpointTest >> testInstall [
 	
-	wp := VariableBreakpoint newWatchPointInClass: VariableBreakpointMockSubclass.	
+	wp := VariableBreakpoint newVariableBreakpointInClass: VariableBreakpointMockSubclass.	
```

Comment:
Can it be simple #newInClass: ? It is already VariableBreakpoint. Why duplicate this meaning in the method name?
And I think #new**For**Class: sound better because the method does not perform installation while  #newInClass: sounds like it does.

Reply:
I went for `newForClass`